Add interlace option to attachment doc

Teranode opened the pull request.

LukeTowers commented:
Does this option work in the thumbOptions array as well for fileuploads? https://github.com/octobercms/docs/blob/993f0ad5ac926a60c59e8e6caa14ba87711d55d3/backend-forms.md#file-upload

If so, could you add it to the documentation as well?

Teranode commented:
Just updated the pull request for the interlacing option for thumbnail generation and added it to the doc, thx for the heads up.

https://github.com/octobercms/docs/pull/245
